### ISSUE TITLE
Order reversed 

### ISSUE THREAD

**Issue by jessuni** [iliakan/javascript-tutorial-en#555]
In this chapter [modifying document](https://javascript.info/modifying-document#tasks), all the order of tasks and its solution buttons are reversed.

**Comment by iliakan** [comment]
What you mean? :)

**Comment by jessuni** [comment]
Is this recently modified as a feature? Now for all tasks in the whole tutorial, the solution button shows up before the content of each task. 
![image](https://user-images.githubusercontent.com/41285428/47070353-95941b80-d223-11e8-97a7-dd0ef6a5763e.png)

**Comment by jessuni** [comment]
No, but the previous version, where tasks are placed before solutions, is more natural in terms of user experience.
